### PR TITLE
Fix register_view.description of multi-dimensional banks

### PR DESCRIPTION
**Pull request by JonatanWaern**

### CODE REVIEW

File: lib/1.4/dml-builtins.dml [hunk 1]
```diff
@@ -1359,28 +1359,43 @@ template bank is (object, shown_desc, _qname) {
...
+            local uint64 num = 0;
+            foreach r in (this._each_register) {
+                if (r.offset != unmapped_offset) {
+                    _reginfo_cache[num] = r;
+                    num++;
                 }
                 num++;
             }
         }
         throw;
```
Comment (lwaern-intel) [inline lib/1.4/dml-builtins.dml:1384]: This `throw` statement doesn't belong.

**Review by lwaern-intel — CHANGES_REQUESTED** [review]
Looks sane enough except for a stray `throw`.

File: lib/1.4/dml-builtins.dml (hunk 1 above)
Comment (JonatanWaern) [reply lib/1.4/dml-builtins.dml]: fixt. mustve snuck in while migrating patches

**Comment by JonatanWaern** [comment]
Accidentally force-pushed a non-rebased version, which seems to break the pull request unrecoverably (forcepushing new, rebased, version doesnt fix). Will dupe to new pull request

**Pull request closed without merging**